atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -107,73 +110,89 @@ public static SerializableTransactionManager createInMemory(Schema schema) {
      */
     public static SerializableTransactionManager create(
             AtlasDbConfig config,
+            java.util.function.Supplier<AtlasDbRuntimeConfig> runtimeConfig,
             Set<Schema> schemas,
             Environment env,
             boolean allowHiddenTableAccess) {
-        log.info("Called TransactionManagers.create on thread {}. This should only happen once.",
-                Thread.currentThread().getName());
-        return create(config, schemas, env, LockServerOptions.DEFAULT, allowHiddenTableAccess);
+        log.info("Called TransactionManagers.create with live reloading config on thread {}."
+                        + " This should only happen once.",
+                UnsafeArg.of("thread name", Thread.currentThread().getName()));
+        return create(config, runtimeConfig, schemas, env, LockServerOptions.DEFAULT, allowHiddenTableAccess);
     }
 
     /**
-     * Create a {@link SerializableTransactionManager} with provided configuration, a set of
+     * Create a {@link SerializableTransactionManager} with provided configurations, a set of
      * {@link Schema}s, {@link LockServerOptions}, and an environment in which to register HTTP server endpoints.
      */
     public static SerializableTransactionManager create(
             AtlasDbConfig config,
+            java.util.function.Supplier<AtlasDbRuntimeConfig> runtimeConfig,
             Set<Schema> schemas,
             Environment env,
             LockServerOptions lockServerOptions,
             boolean allowHiddenTableAccess) {
-        return create(config, schemas, env, lockServerOptions, allowHiddenTableAccess, UserAgents.DEFAULT_USER_AGENT);
+        return create(config, runtimeConfig, schemas, env, lockServerOptions, allowHiddenTableAccess,
+                UserAgents.DEFAULT_USER_AGENT);
     }
 
     public static SerializableTransactionManager create(
             AtlasDbConfig config,
+            java.util.function.Supplier<AtlasDbRuntimeConfig> runtimeConfig,
             Set<Schema> schemas,
             Environment env,
             LockServerOptions lockServerOptions,
             boolean allowHiddenTableAccess,
             Class<?> callingClass) {
-        return create(config, schemas, env, lockServerOptions, allowHiddenTableAccess,
+        return create(config, runtimeConfig, schemas, env, lockServerOptions, allowHiddenTableAccess,
                 UserAgents.fromClass(callingClass));
     }
 
     private static SerializableTransactionManager create(
             AtlasDbConfig config,
+            java.util.function.Supplier<AtlasDbRuntimeConfig> runtimeConfig,
             Set<Schema> schemas,
             Environment env,
             LockServerOptions lockServerOptions,
             boolean allowHiddenTableAccess,
             String userAgent) {
+
+        java.util.function.Supplier<AtlasDbRuntimeConfig> wrappedRuntimeConfig = () -> {
+            validateRuntimeConfig(runtimeConfig.get());
```

Comment:
Agreed. The solutions that I see for this are:
- move it outside the supplier. This would make it log once, on startup, if the runtime config is not there.
- add some logic to prevent this from logging every single time, as in, just log once every 10m.

I will go for the first option, since it seems simpler and we don't want to overcomplicate this.